.github/workflows/plugin_tests.yaml

```diff
@@ -0,0 +1,46 @@
+name: Remote Plugin Tests
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    timeout-minutes: 45
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, windows-latest, macos-latest]
+        pyv: ["3.10"]
```

Comment:
What do you think about using 3.11? 🙂

Reply:
My thinking here was if we are only testing against one python version, 3.10 makes more sense since 3.10 is the version used in our binary pyinstaller builds

Reply:
Okay, makes sense.